backend/shared/shared/oidc/views/eauth_views.py

```diff
@@ -92,10 +97,22 @@ class EauthAuthenticationCallbackView(View):
     http_method_names = ["get"]
 
     def login_success(self):
-        return HttpResponseRedirect(settings.LOGIN_REDIRECT_URL)
+        url = settings.LOGIN_REDIRECT_URL
+
+        lang = self.request.session.get("django_language")
```

Comment:
I believe the default `LANGUAGE_SESSION_KEY` is `_language`, and `LANGUAGE_COOKIE_NAME` is `django_language`. So if you still want to use session, maybe use `_language`  or  `from django.utils.translation import LANGUAGE_SESSION_KEY`

Reply:
Good point! I should've looked more into this.

Reply:
Seems that `LANGUAGE_SESSION_KEY` is deprecated and the cookie should be used.

https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.translation.LANGUAGE_SESSION_KEY:
> The language won’t be stored in the session in Django 4.0. Use the LANGUAGE_COOKIE_NAME cookie instead.

I'll change to that instead.